packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx

```diff
@@ -124,7 +127,7 @@ export const ChatMessageListItem = (props: ChatMessageListItemProps) => {
       })}
     >
       <div className={styles.bubble}>
-        {!isTrackOrPlaylistLink
+        {!showTile
           ? links
```

Comment:
I think we should use `links` and make the `map` into a switch/case or if/else sort of deal?

Reply:
yes!